### DISCUSSION TITLE
Unable to vote and group since taking 4/6 update

### DISCUSSION THREAD

**Discussion by StacyA1013** [microsoft/vsts-extension-retrospectives#541] [Q&A]
One user is unable to vote or group items in the retrospective since we took the 4/6 update (note other users can still use this functionality without issue).  This user is also the creator of boards he cannot use this functionality on.  Any thoughts or troubleshooting we can do to figure out why.  License is professional, is a member of the team.  thank you

**Comment by dphanson63** [comment]
Assuming no longer an issue.
